examples/dao/README.md

```diff
@@ -76,22 +76,6 @@ Below diagram describes the flow of DAO application.
 
 <img src="./docs/DAO-Flow.svg" height="600" width="500" title="DAO flow" />
 
-## Setup Sigma Dao
-
-Setup the sigma_daos table. Indexer should be running before executing below script.
-
-Create a sigma dao user, if not already created:
-
```

Comment:
why do we delete it here?

Reply:
It's not needed. It has been moved to SigmaDAO. Instead of creating separate PR, I have made changes here.